test/src/calib/test_cameracalibration.py

```diff
@@ -9,6 +9,7 @@
  @details Unit tests generated using PyTest
```

Comment:
performance testing is not part of this PR. Should remove and include this in seperate PR that looks specifically at performance testing

Reply:
Done